AK/LexicalPath.h

```diff
@@ -10,6 +10,12 @@
 #include <AK/String.h>
 #include <AK/Vector.h>
 
+// On Linux distros that use glibc `basename` is defined as a macro that expands to `__xpg_basename`, so we undefine it
```

Comment:
Having a hard time believing this as I've been building Lagom on a glibc Linux just fine for ages.

Reply:
My bad, I blatantly copied the comment, and I forgot to update it to say that I only observed this while targeting mlibc. I'll update both comments to reflect this properly. Thanks for catching this!